aws-credentials/tasks/main.yml

```diff
@@ -0,0 +1,16 @@
+---
+- name: Create a dir
+  file:
+    path: /home/ubuntu/.aws/
+    state: directory
+    owner: ubuntu
+    group: ubuntu
+    mode: 0777
+- name: Copy the aws ID and key into a file
+  copy:
+    content: |
+      [restricted_account]
```

Comment:
This should be configurable, and likely just default to "default" - anywhere its not going to be default we can override in group vars